modpack_format/modpack_format.md

```diff
@@ -0,0 +1,89 @@
+# MODIP Modpack Format
+
+The MODIP Modpack Format exists as a simpler alternative to the MODIP Index Format.
+
+## Fields
+
+### `formatType`
+The type of the format. MUST be set to `modipModpack` for the MODIP Modpack Format.
+
+### `formatVersion`
+The version of the format. The current value at the time of writing is `1.0.0`.
+
+---
+
+### `id`
+A unique identifier for this modpack. This field may contain any printable ASCII character except for spaces (U+0021 to U+007E, inclusive)
+
+---
+
+### `name`
+Human-readable name of the modpack.
+
+---
+
+### `summary` (optional)
+A short description of this modpack.
+
+---
+
+### `description` (optional)
+A longer description of the modpack. TODO: Formatted using HTML/subset?
+
+---
+
+### `updates` (optional)
+A URL that can be fetched to retrieve information about updates to the modpack. This follows the update format as specified in **update_format.md**.
+
+---
+
+### `releaseDate`
+The release date of this specific version of the modpack. It MUST be stored as an ISO-8601 conforming string. This MUST include UTC time at the end. A valid example is `2020-01-01T12:00:00Z`. This example date is the 1st of January, 2020, at 12:00:00 UTC. Spaces and time zone offsets (e.g. `+01:00`) CANNOT be used. The `Z` at the end of the string MUST be included and capitalized. The `T` separating the date and time MUST be included and capitalized.
+
+Other values, such as `2020-W32` are NOT allowed. The only allowed format is demonstrated in the example.
+
+### `dependencies`
+The dependencies array contains a list of dependencies that must be downloaded in order for the modpack to be installed. Each object in this array contains the following fields
+
+#### `id`
+A unique identifier for this dependency. See **Standard IDs** for standardized IDs that should be used for certain projects.
+
+#### `version`
+The version of the dependency to be installed.
+
+#### `updates` (optional)
+A URL that can be fetched for updates to the mod. Similar to the modpack's `updates` field. This follows the update format as specified in **update_format.md**.
+
+#### `files` (optional)
```

Comment:
Why is the association between files and dependencies important? i.e. why isn't there just one big files array?

Reply:
Good point, I might do that. But how should non-files (e.g. modloaders) be specified then?

Reply:
What about `"dependencies": {"forge": {...}, "fabric-loader": {...}, "minecraft": {...} /* nothing else, mods are just files */}`?

Reply:
I'm slightly confused, do you mean there should be both a `dependencies` and `files` field? `dependencies` for non-files like modloaders and Minecraft, and `files` for files?

Reply:
Yes that's what I meant

Reply:
Resolved